utils/build.gradle

```diff
@@ -1,18 +1,50 @@
-apply plugin: 'info.solidsoft.pitest'
```

Comment:
The pitest plugin needs to be applied for the mutation tests to work.

Reply:
done

Reply:
Confirmed